tests/readme.md

```diff
@@ -0,0 +1,82 @@
+# Testing
+
+> Refer to the project's `Makefile` for all available options
```

Comment:
Ah, I added this with the intention of deleting line 7 but after re-reading your description I think what you had is better. Let's remove this line and keep what you had 😌 Sorry for the confusion

Reply:
Sure, I'll pull the description there out and just leave the current line 7 in place.